### ISSUE TITLE
Status of CellulaRoBERTa

### ISSUE THREAD

**Issue by thusithathilina** [SyNSec-den/hermes-spec-to-fsm#2]
This is really interesting work and kudos to make it public :) Just wondering whether are you plan to publish the pretrained CellulaRoBERTa model

**Comment by ishtiaqniloy** [comment]
Hi. Thanks a lot for reaching out. This is the link to `CellulaRoBERTa`:
https://drive.google.com/file/d/1R3A5zfM9z6aQzILrfh7aPkSzlQebu5iX/view?usp=sharing

This commit clarifies the link:
https://github.com/SyNSec-den/hermes-spec-to-fsm/commit/7912b61257484ba7637d5134d9293efb6e4386c6

**Comment by thusithathilina** [comment]
Thanks for the clarification. Just another question. How did you (which technique) pre-train the RoBERTa? Was it using mask language modeling? or something else?

**Comment by thusithathilina** [comment]
Found it just checking the config :)
